tests/test_retry.py

```diff
@@ -6,20 +6,27 @@
 #
 
 from __future__ import print_function
-from retry_decorator import *
+from retry_decorator import retry
 
 
-@retry(Exception, tries = 3, timeout_secs = 0.1)
+@retry(Exception, tries=3, timeout_secs=0.1)
 def test_retry():
     import sys
-    print('hello', file = sys.stderr)
+    print('hello', file=sys.stderr)
     raise Exception('Testing retry')
 
+@retry(AttributeError, RuntimeError, tries=3)
+def test_multiple_exceptions():
+    print('hi')
+    raise AttributeError('Testing retry with multiple exceptions')
```

Comment:
Can you create a separate test that tests what happen when position 2 or 3 is raised (i.e. `RuntimeError`) here?